rrat/patients/views.py

```diff
@@ -53,24 +50,42 @@ def patients_list(request):
 
     return render(request, 'patients/patients_list.html', context)
 
+
 @login_required(login_url='users:login')  # Protect the view_patient view
 @check_patient_hidden
 def view_patient(request, id):
     """
     Patient page to view patient details.
     """
     patient = get_object_or_404(PatientModel, id=id)
-
-    # Handle retina photo upload
     form = upload_retina_photo(request=request, patient=patient)
+    images = RetinaPhoto.objects.filter(patient=patient).order_by('-date_created')
 
     context = {
         "form": form,
-        "patient": patient
+        "patient": patient,
+        "images": images,
     }
 
     return render(request, 'patients/view_patient.html', context)
 
+@login_required(login_url='users:login')
+def patient_images(request, id):
+    """
+    Retrieve all retina images associated with a specific patient and display them.
+    """
+    # Get the patient by ID
+    patient = get_object_or_404(PatientModel, id=id)
+
+    # Retrieve all retina images for the patient, ordered by most recent
+    images = RetinaPhoto.objects.filter(patient=patient).order_by('-date_created')
+
+    # Pass images and patient information to the template
+    return render(request, 'patients/view_patient.html', {
+        'patient': patient,
+        'images': images,
+    })
+
```

Comment:
Question, is there a reason this was created? I like the idea of creating a new function to break it out of the view patient function to make things a bit cleaner, but currently this is for a completely different page. We should revise this to return the images, and then call it in the patient_view view function since we have all the images displayed on the view. 

Unless you are thinking of making a new page completely to view the scans. Then we will keep this as is and we will need to revise the patient_view instead. Let me know your thoughts in slack